src/main/java/io/zold/api/RtTransaction.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.zold.api;
+
+import java.time.ZonedDateTime;
+
+/**
+ * RtTransaction.
+ *
+ * @author Izbassar Tolegen (t.izbassar@gmail.com)
+ * @version $Id$
+ * @since 0.1
+ */
+final class RtTransaction implements Transaction {
```

Comment:
@t-izbassar I will have to disagree with @llorllale here, we should call it `StringTransaction`. `Rt-` prefix is usually used for classes that source from a REST API.

@llorllale your call ultimately, what do you think?

Reply:
@carlosmiranda I've seen `Rt` used to stand for "Runtime" (see `amihaiemil/docker-java-api`). That's what I was going for here.

I dislike `StringTransaction` because these days I generally try to avoid compound names (as in, more than one noun).

Let's stick with `RtTransaction` for now. If we need to "source" transactions from somewhere else then we could rename to `TransactionOf` then.

Reply:
@llorllale in that case it's good to merge